client/components/formatted-header/index.jsx

```diff
@@ -19,7 +19,7 @@ function FormattedHeader( { headerText, subHeaderText } ) {
 	} );
 
 	return (
-		<header className={ classes }>
+		<header id={ 'formatted-header' } className={ classes }>
```

Comment:
Would we never have more than one formatted-header on a page?

Reply:
Feels like we can pass this in as a prop

Reply:
We don't have any cases of more than one `FormattedHeader` on a page, but it's not future-proof. Updated on ec1f236b33aef21433712a6afa69816fc7d6077d to get it as a `prop` from the `StepWrapper`.